sw/device/lib/crypto/drivers/hmac.c

```diff
@@ -65,52 +65,57 @@ OT_ASSERT_ENUM_VALUE(HMAC_DIGEST_14_REG_OFFSET, HMAC_DIGEST_13_REG_OFFSET + 4);
 OT_ASSERT_ENUM_VALUE(HMAC_DIGEST_15_REG_OFFSET, HMAC_DIGEST_14_REG_OFFSET + 4);
 
 /**
- * The following values are the only key sizes supported natively bw HWIP.
+ * The following values are the only key sizes supported natively by HMAC HWIP.
```

Comment:
nit: Please update this comment. The key sizes are no longer defined below.

Reply:
There was a small bug with oneshot calls. As I was fixing that I also removed this comment.